src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -847,6 +847,7 @@ def update_auth_settings(cmd, resource_group_name, name, enabled=None, action=No
                          client_id=None, token_store_enabled=None, runtime_version=None,  # pylint: disable=unused-argument
                          token_refresh_extension_hours=None,  # pylint: disable=unused-argument
                          allowed_external_redirect_urls=None, client_secret=None,  # pylint: disable=unused-argument
+                         client_secret_certificate_thumbprint=None,  # pylint: disable=unused-argument
```

Comment:
May I know why it is unused?

Reply:
It's sent to the app as an authSetting. I don't know why it's unused, but it's no different than the other variables which have the unused tag next to them, so I was just trying to make it match up. What does the unused argument tag mean?

Reply:
Pylint does not allow unused argument. The comment disables this rule.